yarn.lock

```diff
@@ -9364,20 +9393,30 @@ react-dom@16.13.1, react-dom@^16.8.6:
     prop-types "^15.6.2"
     scheduler "^0.19.1"
 
+react-dom@^16.14.0:
```

Comment:
It looks like we have a duplicate, because the deps or our deps have 16.13.1 without the `^` 🤔 Maybe we could downgrade to 16.13.1 without `^` to avoid duplication?

Reply:
On the other hand all the other deps that are using older version are tools for development & testing, so the end repo (the one where we use bubble-dev) shouldn't bundle double the `react-dom` 🤔
Ok, I think it's ok. Let's 🚀